src/NzbDrone.Common/Disk/DiskTransferService.cs

```diff
@@ -511,6 +502,27 @@ private void TryMoveFileVerified(string sourcePath, string targetPath, long orig
             }
         }
 
+        private void VerifyFile(string sourcePath, string targetPath, long originalSize, string action)
+        {
+            var targetSize = _diskProvider.GetFileSize(targetPath);
+
+            if (targetSize == originalSize)
+            {
+                return;
+            }
+
+            _logger.Debug("File {0} incomplete, waiting in case filesystem is not synchronized. [{1}] was {2} bytes long instead of the expected {3}.", action, targetPath, targetSize, originalSize));
```

Comment:
Extra bracket

```suggestion
            _logger.Debug("File {0} incomplete, waiting in case filesystem is not synchronized. [{1}] was {2} bytes long instead of the expected {3}.", action, targetPath, targetSize, originalSize);
```